internal/xds/translator/httpfilters.go

```diff
@@ -100,26 +100,30 @@ func newOrderedHTTPFilter(filter *hcmv3.HttpFilter) *OrderedHTTPFilter {
 	switch {
 	case isFilterType(filter, egv1a1.EnvoyFilterHealthCheck):
 		order = 0
-	case isFilterType(filter, egv1a1.EnvoyFilterFault):
+	case isFilterType(filter, egv1a1.EnvoyFilterGRPCWeb):
```

Comment:
shouldnt these be added much later in the chain, closer/after RL ?

Reply:
Yeah, makes sense. Putting it after RL can avoid unnecessary transcoding.